bin/partial_upload.py

```diff
@@ -0,0 +1,76 @@
+#!/bin/env python
+# -*- coding: utf8 -*-
+""" Triggers a partial upload process with the specified raw.xz URL. """
+
+import argparse
+import logging
+import logging.config
+
+import fedmsg.config
+
+from fedimg.config import AWS_ACCESS_ID
+from fedimg.config import AWS_SECRET_KEY
+from fedimg.config import AWS_BASE_REGION, AWS_REGIONS
+from fedimg.services.ec2.ec2copy import main as ec2copy
+from fedimg.services.ec2.ec2initiate import main as ec2main
+
+logging.config.dictConfig(fedmsg.config.load_config()['logging'])
+log = logging.getLogger('fedmsg')
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description="Trigger a partial upload based on the arguments")
+    parser.add_argument(
+        "-u", "--url", type=str, help=".raw.xz URL", required=True)
+    parser.add_argument(
+        "-c", "--compose-id", type=str, help="compose id of the .raw.xz file",
+        required=True)
+    parser.add_argument(
+        "-p", "--push-notifications",
+        help="Bool to check if we need to push fedmsg notifications",
+        action="store_true", required=False)
+    parser.add_argument(
+        "-v", "--volume", help="volume type for the image", required=False)
+
```

Comment:
if you decide to use the logger, you could maybe have a --debug flag to trigger the debug log level

Reply:
I think it's better to remove the logger itself.